site/docs/v1/tech/core-concepts/users.adoc

```diff
@@ -112,3 +118,15 @@ It is possible, though rare, for an Elasticsearch index to become out of sync wi
 However, in general, even if a temporary outage occurs with Elasticsearch, the index will be sync up automatically. Reindexing is an expensive operation, especially if your system has a large number of users, so it should not be run unless necessary.
 
 If you do need to run this, navigate to [breadcrumb]#System -> Reindex# in the FusionAuth admin UI to initiate a reindex of all users. This navigation item will only be displayed when the search engine is Elasticsearch.
+
+== User localization
+
+FusionAuth is built from the ground up with localization in mind. You can assign a user's `preferredLanguages` via the API or link:/docs/v1/tech/guides/advanced-registration-forms/[advanced registration form]. and add that value to a JWT or retrieve it from via the link:/docs/v1/tech/apis/users[User API]. Each application registration can also have a preferred language associated with it.
+
+Both the hosted login pages and the emails can be localized; the former via link:/docs/v1/tech/themes/localization/[themes] and the latter via link:/docs/v1/tech/email-templates/email-templates/[email templates].
+
+If you set the user's preferred language, FusionAuth's administrative user interface will localized. There are no translations of the FusionAuth administration user interface, however. 
+
+For instance, in the below screenshot, Richard's account has a `preferredLanguage` of `French`. When Richard interacts with the FusionAuth administrative user interface,  the account with the email dinesh@fusionauth.io, which was created on Oct 5, 2020, has a creation date display formatted as specified by the French locale, with the day first.
```

Comment:
Not sure we need to mention this or not.. but we currently support the following locales as it relates to formatting date and time strings.  Where the suffix is the locale 

```
en
ar
cs
da
de
el
en
en_US
es
fr
ga
he
hi
it
ja
ko
nl
no
pt
ru
sv
zh
zh_CN
zh_TW
```
( https://github.com/FusionAuth/fusionauth-app/tree/691f9f196a390205d027aec284d164a91bb04085/web/WEB-INF/messages see `package.*` in this directory. 

These are easy to add, all we keep are the date and time format strings. For example, for French (`fr`) we have

```
date-format=d/M/yyyy
date-time-format=d/M/yyyy hh:mm a z
date-time-seconds-format=d/M/yyyy hh:mm:ss a z
```

So if anyone finds that their preferred locale is not producing the date and time formats as they expect they should open an issue and we'll add it.